### PR TITLE
Changed gurobi from external to core

### PR DESCRIPTION
**Pull request by batatasfritas**

### CODE REVIEW

File: configuration/packages.yaml [hunk 1]
```diff
@@ -95,10 +95,10 @@ packages:
...
       likwid@4.3.0 %gcc@4.8.5: /ssoft/spack/external/LIKWID/4.3.0
 
-  gurobi:
-    buildable: False
-    paths:
-      gurobi@8.1.1%gcc@4.8.5: /ssoft/spack/external/gurobi/8.1.1
+  # gurobi:
+  #   buildable: False
+  #   paths:
+  #     gurobi@8.1.1%gcc@4.8.5: /ssoft/spack/external/gurobi/8.1.1
```
Comment (rmsds) [inline configuration/packages.yaml]: I think we should actually remove these lines.

**Review by rmsds — CHANGES_REQUESTED** [review]
Looks good, just needs the obsolete lines from `packages.yaml` to be deleted.

**Comment by nrichart** [comment]
@rmsds this also should be merge for https://github.com/epfl-scitas/spack/pull/254

**Review by rmsds — DISMISSED** [review]

**Review by nrichart — APPROVED** [review]
lgtm

**Pull request merged**